### PR TITLE
Fix document formatting

### PR DESCRIPTION
**Pull request by yuewko**
Fix formatting issue for README.md.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Prerequisite
 
 Install Dependency
 ------------------
-The driver primarily depend on libnetwork. This can 'supposedly' be installed using the following command:
+The driver primarily depend on libnetwork and the infoblox-go-client . They can be installed using the
```
Comment (johnbelamaric) [inline BUILD.md]: "depends on"

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ docker run -v /var/run:/var/run -v /run/docker:/run/docker ./infoblox-ipam --gri
 Note that the -v options are necessary to provide the container access to the specified directories on the
 host file system.
 
-For coneniences, a script called "run-container.sh" is proivded.
+For convenience, a script called "run-container.sh" is proivded.
```
Comment (johnbelamaric) [inline README.md]: s/proivded/provided/

**Comment by yuewko** [comment]
Made the above changes. Let me know if that's all and I'll push it.

**Comment by johnbelamaric** [comment]
yes, that is all

On Feb 12, 2016, at 9:39 AM, yuewko <notifications@github.com<mailto:notifications@github.com>> wrote:

Made the above changes. Let me know if that's all and I'll push it.

**Pull request merged**